src/utils/helpers/MappingHelpers.ts

```diff
@@ -86,3 +87,27 @@ export function filterConnectedNodes(network: Network) {
             network.links.findIndex((link: DependencyLink) => compareNodes(link.source, node) || compareNodes(link.target, node)) >= 0
     );
 }
+
+export function mapServicesToTreeNodes(services: Service[]): TreeNode[] {
+    const allNodes: TreeNode[] = services.map(service => ({
+        name: service.name,
+        consumers: [],
+        providers: [],
+    }));
+
+    return services.reduce((nodes: TreeNode[], service: Service) => {
```

Comment:
wouldn't it be simpler to create empty allNodes array and just push to it on in forEach on services ?

Reply:
Sure, looks better to me